packages/manager/modules/billing/src/payment/method/add/component.js

```diff
@@ -17,5 +17,6 @@ export default {
     model: '<',
     onPaymentMethodAdded: '<',
     paymentMethods: '<',
+    isDisplayableIndiaCreditCardInfoBanner: '<',
```

Comment:
**nitpick:** Since in the feature flipping we give opportunity to add other subsidiaries apart from `India`. maybe this variable can be renamed as it will not be just restricted to India.
```suggestion
    displayCreditCardInfoBanner: '<',
```

Reply:
Done Anoop Thanks !!